cypher-shell/src/integration-test/java/org/neo4j/shell/MainIntegrationTest.java

```diff
@@ -16,39 +17,47 @@
 
 public class MainIntegrationTest {
 
-    @Test
-    public void connectInteractivelyPromptsOnWrongAuthentication() throws Exception {
+    private String inputString = String.format( "neo4j%nneo%n" );
+    private ByteArrayOutputStream baos;
+    private ConnectionConfig connectionConfig;
+    private CypherShell shell;
+    private Main main;
+
+    @Before
+    public void setup() {
         // given
-        // what the user inputs when prompted
-        String inputString = String.format( "neo4j%nneo%n" );
-        InputStream inputStream = new ByteArrayInputStream(inputString.getBytes());
+        InputStream inputStream = new ByteArrayInputStream( inputString.getBytes() );
 
-        ByteArrayOutputStream baos = new ByteArrayOutputStream();
-        PrintStream ps = new PrintStream(baos);
+        baos = new ByteArrayOutputStream();
+        PrintStream ps = new PrintStream( baos );
```

Comment:
I don't really care about this but this is neo4j styling not cypher-shell styling.

Reply:
There is no cyher-shell styling. No enforced one and no consistent one across the repository.